software/contrib/logic.md

```diff
@@ -0,0 +1,41 @@
+# Logic
+
+This program implements six digital logic operations, with each
+operation's result sent to a different output jack.
+
+- `ain`: the first boolean input; any signal above 0.8V is treated
+  as ON, anything below that is OFF. (Note: this is the same voltage
+  threshold used on the digital input.)
+- `din`: the second boolean input
+- `cv1`: `ain` AND `din`
+- `cv2`: `ain` OR `din`
+- `cv3`: `ain` XOR `din`
+- `cv4`: `ain` NAND `din`
+- `cv5`: `ain` NOR `din`
+- `cv6`: `ain` XNOR `din`
+
+The outputs are as follows:
+
+| Output No. | Operation |
+|------------|-----------|
+| Out 1      | AND       |
+| Out 2      | OR        |
+| Out 3      | XOR       |
+| Out 4      | NAND      |
+| Out 5      | NOR       |
+| Out 6      | XNOR      |
+
+The following table shows the value for each operation for every
+input combination:
+
+| Ain | Din | AND | OR | XOR | NAND | NOR | XNOR |
```

Comment:
[optional] nitpick, but I would swap Ain and Din since that is their order on the EuroPi.